README.rst

```diff
@@ -176,46 +195,173 @@ For flashing the board OpenOCD must be used. You can optionally use gdb
 as a frontend for OpenOCD as described below.
 
 You must first flash a bootstrap rom before flashing an application.
-Assuming the toolchain was unpacked into *$HOME/issm_2016.0.019/*, this can be
+Assuming the toolchain was unpacked into *$HOME/issm_2016/*, this can be
 done with:
 
-``$ cd $HOME/issm_2016.0.019/tools/debugger/openocd``
+``$ cd $HOME/issm_2016/tools/debugger/openocd``
 
 ``$ ./bin/openocd -f scripts/board/quark_d2000_onboard.cfg``
```

Comment:
One last thing :) Can we include the OpenOCD config file for the SE as well --- quark_se_onboard.cfg...

The rest looks fine.